openapi/components/schemas/Profile.yaml

```
@@ -71,3 +71,6 @@ properties:
       User preferences like timezone, language and many more. This is an object
       with custom properties.
     type: object
+  isPermissionsEmulation:
```

```suggestion
  hasPermissionsEmulation:
```
I'm looking for something that is grammatically correct?